tools/lint.ts

```diff
@@ -0,0 +1,24 @@
+console.log("clippy");
+
+await Deno.run({
+  cmd: [
+    "cargo",
+    "clippy",
+    "--all-targets",
+    "--release",
```

Comment:
`--release` should be used only if `--release` arg is passed to this script - I run clippy frequently locally and I just want to run debug build (because it's faster). CI should still run `--release`